projects/exui-common-lib/src/lib/gov-ui/components/hmcts-banner/hmcts-banner.component.ts

```diff
@@ -0,0 +1,17 @@
+import {Component, Input} from '@angular/core';
+
+/*
+* Hmcts Banner
+* Responsible for displaying prominent message and related actions
+* @prop message to display
+* @prop type
+* */
+@Component({
+  selector: 'xuilib-hmcts-banner',
+  templateUrl: './hmcts-banner.component.html'
```

Comment:
missing scss file. It still needs some styles for the element itself (like setting display to block, and width to 100% by default, as it's an unknown element so it works in all browsers)